web/source/kmwembedded.js

```diff
@@ -450,7 +450,7 @@
       if(!keymanweb._ActiveKeyboard ||  Lkc.Lcode == 0) return false;
       
       // If key is mapped, return true
-      if(keymanweb._ActiveKeyboard['gs'](Lelem, Lkc)) return true;
+      if(keymanweb.processKeystroke(util.physicalDevice, Lelem, Lkc)) return true;
```

Comment:
Should this be `util.physicalDevice` or `util.device`? These are popup key events so they are touch-based, right?

Reply:
Good catch.  I was too caught up with all the 'physical' changes that I blanked out on that detail.